src/test/java/com/hubspot/ringleader/watcher/PersistentWatcherTest.java

```diff
@@ -190,7 +191,7 @@ public Thread newThread(Runnable r) {
       }
     });
 
-    WatcherFactory factory = new WatcherFactory(curatorSupplier, executor);
+    WatcherFactory factory = new WatcherFactory(curatorSupplier, executor, TimeUnit.MINUTES.toMillis(1));
```

Comment:
I only made this single change to the existing tests to (try to) make sure I didn't break anything.